ops-bedrock/docker-compose.yml

```diff
@@ -11,6 +11,14 @@ volumes:
 
 
 services:
+  da:
+    container_name: da
+    platform: linux/x86_64
+    image: "ghcr.io/celestiaorg/local-celestia-devnet:main"
```

Comment:
should we lock this on a version? i.e. `v0.9.1` which is latest?

Reply:
we also need to change this to `rollkit/local...`

Reply:
Done.